deploy/config.yaml.example

```diff
@@ -5,6 +5,7 @@ rps:
   configuration:
     databaseAccountName: ''
     keyvaultPrefix: ''
+    storageAccountName: ''
```

Comment:
aside - I don't see the point of this file and would be have no regrets if you wanted to delete it.  If you do that please remove it from the generate.go too